src/components/HeaderBar.tsx

```diff
@@ -4,7 +4,7 @@ import { Link } from 'react-router-dom';
 
 export default function HeaderBar() {
 	return (
-		<div className="flex justify-between w-full p-4 shadow-headerbar mb-3">
+		<div className="flex mb-2 justify-between w-full p-4 shadow-headerbar">
```

Comment:
@ElijahMSmith changed to mb-2 bc mb-3 was hiding the shadow when placing something under it. I think this may bw because the shadow was only 2PX tall.